### ISSUE TITLE
Nominations Open: Community Board Seat for OpenJS Foundation Board of Directors

### ISSUE THREAD

**Issue by jorydotcom** [openjs-foundation/cross-project-council#616]
**Nominations are now open for the OpenJS Board for Secondary CPC Director, a community representation seat.** OpenJS Foundation project community members are encouraged to run for this position on our Board of Directors. 
 
**Nomination period / Election process** 
Nominations will be taken from August 10 through August 16. Voting will be open August 17-21 with new Directors notified on Aug 24. Term begins Sept. 1, 2020 and concludes Aug. 31, 2021.

CPC Voting Members will vote to determine who will be elected.
The election will be run using OpaVote, Scottish STV method.

**To Self-Nominate or Nominate another:**
If you are interested in running for election, please reply to this GitHub issue including your name, project affiliation, and why you are interested in running for the Board. 

If you are nominating another, please email their name, project affiliation and why you think they would be a good Board Director to Jory Burson and Joe Sepi. We will confirm their willingness to participate before including them in the list of candidates.

**Background / About the OpenJS Foundation Board of Directors:**
As a non-profit, the OpenJS Foundation is overseen by a Board of Directors. Responsibilities range from managing the business oversight of the Foundation, including budget, marketing, and Foundation staff, to setting legal policy. The Board has a responsibility to ensure that Foundation activities support its purpose as defined in the bylaws:
*   Promote the widespread adoption and continued development of key JavaScript solutions and related technologies worldwide.
*   Facilitate collaboration within the JavaScript and web development community.
*   Create a center of gravity for open source projects throughout the end-to-end JavaScript ecosystem guiding them toward open governance and diverse collaborator bases.
*   Host the infrastructure to support hosted JavaScript open source projects.
*   Enable, through advancement of Projects and strategic partnerships, an open and accessible web.
*   Undertake such other activities as may from time to time be appropriate to further the purposes and achieve the goals set forth above.

There are four ways by which one becomes a Director.
* Your company is a Platinum member, and you are appointed.
* Your company is a Gold or Silver member, and you are elected by the other Gold or Silver members.
* Your company is an end user of OpenJS Foundation projects, and you are elected by the other members to represent the views of end users.
* You are chosen by the community to fill one of the community seats. The Cross Project Council is on point for this process.

**Responsibilities of Directors:**
All Directors have a responsibility to represent the organization or community group which appointed or voted them into their role. In addition, all Directors have a responsibility to represent the best interests of the organization and our development communities. At a high level, a Director’s responsibilities include:
*   Attending as many board meetings as possible. The board generally meets once per month for two hours. Board meetings require quorum in order to vote, and every meeting includes a vote on the prior meeting’s minutes.
*   Be in touch with your constituents. The most effective board members are in tune with the facts of their community, and opinions held by the people they represent. Their objective is to represent these faithfully and articulately in board matters. Community Directors are expected to be active participants in CPC meetings.
*   Be in touch with your legal counsel. The OpenJS Foundation has legal counsel to advise the Foundation itself. However, for matters of policy, Directors should discuss their positions with corporate counsel to ensure they are aligned with their organization’s positions.

For additional information on these roles and their responsibilities, see: 
* https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-8-board-representation
* https://github.com/openjs-foundation/cross-project-council/blob/master/COMMUNITY_BOARD_SEAT_EXPECTATIONS.md

**Comment by joesepi** [comment]
For clarity, here is a direct link to our Charter which details who is eligible for this seat (Active OpenJS Collaborators) and who will vote in this election (Voting CPC members):
https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#the-secondary-cpc-director-as-defined-in-43e-in-theopenjs-foundation-bylaws

**Comment by jasnell** [comment]
Hello. I'm nominating myself for this. I am a member of the Node.js Technical Steering Committee and a long time contributor to Node.js. I (briefly) sat as the Node.js TSC Board representative on the former Node.js Foundation Board. My interest in this seat is to represent the interests of the greater JavaScript open source community and ensure they continue to have an active voice in the evolution of the Foundation. 

Disclosure: I am the Head of Research at NearForm, which is currently a silver level member of the Foundation.

**Comment by mhdawson** [comment]
I'm self nominating myself.

I'm interested serving as the Community Board representative. I'm an active member of the Cross Project Council, am involved in the Node.js project and acted as the Node.js Board representative for the last year.

My goal as a board member would be to:

* bring the perspective of the foundation projects and greater community to the board;
* ensure the needs of foundation projects are considered in the decisions that are made;
* ensure that we communicate information shared with the board with the projects whenever possible;
* when feasible/applicable ensure we work to get input in advance from foundation projects as input to board decisions;
* help champion some of the broader/longer term initiatives that are important to the success of the foundation in the long run. 

While I know the Node.js project well and have learned about a number of the foundation projects through my work in the Cross project council and the recent collaborator summit I'd work to learn more as the Community Board representative to be able to represent the broader community.  One ask that I've heard in the discussion around moving the board seat to be representing the projects across the foundation was updates from the director in the TSC/CommComm meetings which we've done over the past year. I think it would make sense to do that for other projects who were interested and would be happy to work to to find the right way to do that as well.

**Comment by dylans** [comment]
I'm self nominating which I frankly find to be a bit awkward but here it goes. :)

I'm interested in helping serve on the OpenJS Foundation board if my involvement is helpful for the community.

I'm the co-creator of Dojo and the original Dojo Foundation which later merged with the jQuery Foundation to become the JS Foundation, which then merged with the Node.js Foundation to become the OpenJS Foundation. Currently I serve as the Dojo representative for the OpenJS Cross Project Council.

Through these various foundation iterations I've been supporting the concepts of open governance and collaboration for 15 years. My primary motivation for caring about foundations is to help projects and their founders and communities. It's challenging to create, maintain, and grow open source projects. This is true for both foundation projects and really any other open source project that seeks to manage itself following open principles.

My style is to say yes whenever reasonable requests get made, to give clear explanations for when the answer should be no, and to default to being as transparent and open as possible.

I've also run a few companies over the years and while I mostly identify as an engineer, I also spend a lot of time thinking about engineering from a business perspective and creating engineering-driven organizations. I also run HalfStack which is a series of events that explore the creative side of JS+web in a manner that is very focused on community.

If my approach and perspective serves the community and the foundation well then I'm happy to serve on the board. If there's a better candidate at this time given the current needs of the foundation then that would be equally amazing. It's not about me but about how we can continue to support the foundation and its projects and contributions, as well as the broader open JavaScript ecosystem.

Have I mentioned that I'm rather passionate about our community? ;)

**Comment by tobie** [comment]
Hi all,

I care deeply about this community and would _love_ to represent it in the Board.

I have a background in JS and Web development, and spent most of my career bridging software, business, and law (though I'm not a lawyer). I thrive in complex multi-stakeholder environments and have a track-record of relentless focus on contributor experience and consensus building. I'm particularly proud of having enabled the transformation of W3C's rudimentary testing effort into Web Platform Tests by driving critical changes in its contribution policy.

I also deeply care about diversity and inclusion. I waited until the very last minute to nominate myself, hoping there would be a more diverse set of candidates to chose from.

In the absence of a broader set of candidates, we should at the very least delay the end of the nomination period and do some serious outreach to improve diversity.

We owe it to our community.

**Comment by SaraJo** [comment]
I am also self nominating. 

Recently I completed my one year board stint at the .NET Foundation. I learned a ton about governance from my first experience as an open source board member. I chaired the Outreach committee during my tenure, and helped launch the Virtual .NET Meetups, as a resource for our thousands of meetup groups that found themselves without event space. 

I decided to not run for re-election this year as I have been really enjoying the work I've been doing with the CPC to date and wanted to give myself more time for them. I'll be continuing my work with the CPC, board member or not. It feels great to be giving back to a community that has been a resource over many years. 

I have been writing JavaScript for 20 years professionally (crazy!) Frontend and backend. I've been a manager for almost 10 years, sometimes when I try to tell my partner how he should be building something he says "Sara, when is the last time you got paid to code?" and I yell "I'll always be a software engineer!" and then I slam the door. #2020, amirite? At this point in my career I have found the place I can contribute the most is in the world of governance. I've been in leadership at several companies, and have seen the way the cheese gets made enough that it's become valuable experience to share with others. 

As a board member, I think I can best offer a neutral representation of the JavaScript community that isn't explicitly tied to an organization or project. I think this is beneficial as the community at large's needs are different than projects under the foundation, or even the foundation itself. 

Additionally, I feel the the OpenJS board can help with two things that are essential in open source (language agnostic). I'll always be bringing my thoughts and priorities through this lense:

* **Sustainability for projects and project contributors.** Open source burnout is very real, and the reason for a lot of projects sunsetting at the least, and often can lead to anxiety and depression in maintainers. Helping projects reach sustainability is something that I am passionate about, I'll always bring that voice with me. None of us can solve this alone, but when I speak to contributors and project owners that is something that comes up quite often. 

* **Pipelines for new contributors who don't know how to get started.** There are 23 million software developers in the world, and the majority of them haven't contributed to open source. When I talk to new coders, or even experienced coders, I hear from them "I don't know how to get started," and "it's very intimidating." This need is symbiotic with the needs of project owners and can also be a place that we can offer help.
